mqtt/mqtt-bridge/src/pump/egress.rs

```diff
@@ -80,15 +79,15 @@ where
                 }
             })
             .buffer_unordered(MAX_IN_FLIGHT);
-        pin_mut!(load_and_publish);
+        pin_mut!(publications);
```

Comment:
nit: I liked the old name because it signified what was happening.

Reply:
Oops, I guess it is left from my experiments

Reply:
So we are going to leave it?